src/main/scala/org/constellation/API.scala

```diff
@@ -106,6 +107,11 @@ class API(udpAddress: InetSocketAddress)(implicit system: ActorSystem,
         path("channels") {
           complete(dao.threadSafeMessageMemPool.activeChannels.keys.toSeq)
         } ~
+          pathPrefix("data") {
+            path("channels") {
```

Comment:
you can have / in a path, i.e. path("data" / channels"). prefix is only really useful if you have multiple things nested under it

Reply:
Added some more things here so it's ok